.env.example

```diff
@@ -85,6 +85,7 @@ TEMPORAL_POSTGRES_DB=temporal
 TEMPORAL_POSTGRES_USER=temporal
 TEMPORAL_POSTGRES_PASSWORD=<your_temporal_postgres_password>
 TEMPORAL_SCHEDULE_TO_CLOSE_TIMEOUT=3600
+TEMPORAL_SEARCH_ATTRIBUTE_KEY=JulepExecutionId
```

Comment:
it should be `CustomStringField` by default because dev and local ship with that. on prod, we can set this to `JulepExecutionId` (maybe leave a comment to that effect)

Reply:
I was thinking of doing this the other way around (having this by default and setting it to `CustomStringField` in `.env` files).

Reason for that is, for anyone else from the public running this, having this key defaulting to `JulepExecutionId` is clearer than `CustomStringField`.

@creatorrr lmk if that makes sense.

Reply:
but in order for that to work, you'll have to add `JulepExecutionId` to temporal server first from docker compose setup (it's not added on the fly when you use the sdk)

https://docs.temporal.io/search-attribute

https://community.temporal.io/t/custom-searchattributes/2943

![image](https://github.com/user-attachments/assets/0e5da1ac-76d6-470f-99b3-7f5acfd3c3eb)